components/formats-gpl/pom.xml

```diff
@@ -308,43 +305,20 @@
       <id>callan</id>
       <name>Chris Allan</name>
       <email>callan@lifesci.dundee.ac.uk</email>
-      <url>http://www.openmicroscopy.org/site/about/development-teams/jason</url>
+      <url>https://glencoesoftware.com/chris-allan.html</url>
       <organization>Swedlow Lab, University of Dundee</organization>
-      <organizationUrl>http://www.lifesci.dundee.ac.uk/groups/jason_swedlow/</organizationUrl>
+      <organizationUrl>https://glencoesoftware.com</organizationUrl>
```

Comment:
Would it be better to change the organization link text (and email address) too rather than the confusion of the link saying UoD but going to Glencoe?